packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/ExtraFields/template.success.tsx

```diff
@@ -355,6 +355,31 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
     )
   }
 
+  const RenderTextBoxQuestion = (node: NodeTextType) => {
+    const validFormatCb = useCallback(validFormat(node.regex), [node.regex])
+    const validations = [required, validFormatCb]
+    return (
+      <FormGroup>
+        <QuestionTitle>{node.text}</QuestionTitle>
+
+        {node.instructions && node.instructions !== '' && (
```

Comment:
How about declaring a const, for the sake of clarity?

`const displayInstructions = node.instructions && !!node.instructions.length`

...

`{displayInstructions && (`

Reply:
Nice! Thanks